debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresConnectorConfig.java

```diff
@@ -1216,6 +1224,10 @@ public boolean isYbConsistentSnapshotEnabled() {
         return getConfig().getBoolean(YB_CONSISTENT_SNAPSHOT);
     }
 
+    public boolean isYbLoadBalanceConnections() {
```

Comment:
Rename the method - `ybShouldLoadBalanceConnections`.

It is not mandatory that we simply add `is` or `has` kind of words since Debezium itself doesn't follow this standard much. I have noticed that it's more of what makes more sense while reading.